models/comment_test.go

```diff
@@ -0,0 +1,34 @@
+package models
+
+import (
+	"testing"
+)
+
+func TestComment_Validate(t *testing.T) {
+	testCases := []struct {
+		name       string
+		in         Comment
+		errMessage string
+	}{
+		{
+			name: "Validation passed",
+			in: Comment{
+				Content: "content",
+			},
+			errMessage: "",
+		},
+		{
+			name:       "Content is empty",
+			in:         Comment{},
+			errMessage: requiredErrMessage,
+		},
+	}
+
+	for _, tc := range testCases {
+
+		t.Run(tc.name, func(t *testing.T) {
```

Comment:
please read what is function `t.Parralel()`
and after please read this https://gist.github.com/posener/92a55c4cd441fc5e5e85f27bca008721 (but blease read carefully to understand everything), and only after please fix this in your tests.